modules/twinklewarn.js

```diff
@@ -1306,6 +1319,18 @@ Twinkle.warn.callbacks = {
 
 		var date = new Date();
 
+		var vantimestamp = parseInt(mw.util.getParamValue('vantimestamp'));
+		var revDate = new Date(vantimestamp);
+		if (vantimestamp && !isNaN(revDate)) {
+			revDate.setUTCHours(revDate.getUTCHours() + 24);
+			if (revDate < date) {
```

Comment:
```suggestion
			if (revDate.getTime() < date.getTime()) {
```
type-correctness.

Reply:
Quite a few of these throughout...

Reply:
Think I got 'em all?  Eyes go blurry looking over `latest_date` and `history[current[1]]`